pkg/network/config/config_test.go

```diff
@@ -22,6 +22,18 @@ import (
 	aconfig "github.com/DataDog/datadog-agent/pkg/config"
 )
 
+// variables for testing config options
+const (
+	driverDefaultNotificationThreshold = 512
+	driverMaxFragmentLimit             = 512
+	validNotificationThreshold         = 100
+	invalidNotificationThreshold       = 1200
+	invalidHTTPRequestFragment         = 600
```

Comment:
rather than promoting these to package level constants, maybe move them outside the subtests, but still within each `Test*` function? I don't _think_ they are used in multiple.